{
  "id": "CVE-2018-10882",
  "assigner": "secalert@redhat.com",
  "Published": "2018-07-27T16:29:01.407Z",
  "Modified": "2023-02-12T22:31:41.267Z",
  "last-modified": "2023-02-12T22:31:41.267Z",
  "summary": "A flaw was found in the Linux kernel's ext4 filesystem. A local user can cause an out-of-bound write in in fs/jbd2/transaction.c code, a denial of service, and a system crash by unmounting a crafted ext4 filesystem image.",
  "access": {
    "authentication": "NONE",
    "complexity": "LOW",
    "vector": "LOCAL"
  },
  "impact": {
    "availability": "COMPLETE",
    "confidentiality": "NONE",
    "integrity": "NONE"
  },
  "cvss": 4.9,
  "exploitabilityScore": 3.9,
  "impactScore": 6.9,
  "cvss-vector": "AV:L/AC:L/Au:N/C:N/I:N/A:C",
  "references": [
    "http://www.securityfocus.com/bid/106503",
    "https://access.redhat.com/errata/RHSA-2018:2948",
    "https://bugzilla.kernel.org/show_bug.cgi?id=200069",
    "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-10882",
    "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=c37e9e013469521d9adb932d17a1795c139b36db",
    "https://lists.debian.org/debian-lts-announce/2018/07/msg00020.html",
    "https://usn.ubuntu.com/3753-1/",
    "https://usn.ubuntu.com/3753-2/",
    "https://usn.ubuntu.com/3871-1/",
    "https://usn.ubuntu.com/3871-3/",
    "https://usn.ubuntu.com/3871-4/",
    "https://usn.ubuntu.com/3871-5/"
  ],
  "vulnerable_configuration": [],
  "vulnerable_product": [],
  "vendors": [],
  "products": [],
  "vulnerable_product_stems": [],
  "vulnerable_configuration_stems": [],
  "cwe": "CWE-787",
  "vulnerable_configuration_cpe_2_2": []
}